emotion_classification.py

```diff
@@ -0,0 +1,78 @@
+import torch
+
+import gluonnlp as nlp
+import numpy as np
+from torch.utils.data import Dataset, DataLoader
+
+from kobert.utils import get_tokenizer
+from kobert.pytorch_kobert import get_pytorch_kobert_model
+from bert_classifier import BERTClassifier
+
+emotion_clsf_weights_file = "model_weights.pth"
+
+kobert_model, vocab = get_pytorch_kobert_model()
+model = BERTClassifier(kobert_model, dr_rate=0.5)
+model.load_state_dict(torch.load(emotion_clsf_weights_file, map_location=torch.device('cpu')))
+model.eval()
```

Comment:
아마 가장 궁금한 점이 모델을 저장하고 불러오는 이 부분일 거라고 생각해요.

모델을 저장하는 방법은 아래와 같습니다.

colab에서는 모델을 학습시킨 이후에 
즉,
[sojin, 감정 분류 모델 학습 코드에서](https://github.com/Nicer-since-2021/multiclass-emotion-classification-using-KoBERT/blob/main/SOJIN/emotion_classification_kobert.ipynb)
```
#토큰화
tokenizer = get_tokenizer()
```
이 전에 아래의 코드를 추가하여 모델을 저장했습니다.
```
# 마운트 시킨 드라이브에서 모델을 저장하고 싶은 폴더로 이동
%cd '/content/drive/MyDrive/주은문서/model_folder'
# model_weights.pth (`pth`: pytorch model extension) 파일 이름으로 학습시킨 모델의 가중치만을 저장
torch.save(model.state_dict(), 'model_weights.pth')
# 모델 가중치 파일이 저장된 폴더 압축 (압축해도 기존의 360MB 파일이 340MB나 되기 때문에 의미가 없긴 합니다...)
!zip -r /content/drive/MyDrive/주은문서/model_folder.zip /content/drive/MyDrive/주은문서/model_folder/
```

14줄: 모델의 모양까지는 저장을 안했기 때문에 model의 속성들을 알 수 있도록 BERTClassifier 생성자를 이용해 정의해줍니다.
15줄: 그 다음 학습시켜 도출된 가중치 값을 불러오고 
16줄: 모델을 더 학습시키지 않기 때문에 eval time으로 변경하여 evaluation 과정에서 사용하지 않아야 하는 layer들은 off 시켜줍니다.